model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,42 @@
+groups:
+  - id: mobile-crash-event
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'mobile' and `event.name` is 'crash
```

Comment:
I think the domain should be `device` which is the one targeting mobile apps in the [events spec](https://opentelemetry.io/docs/specs/otel/logs/semantic_conventions/events/).